src/components/Icons/IconSvgs.tsx

```diff
@@ -167,6 +169,8 @@ export default {
   utilityAccountFilled,
   utilityAccountUnfilled,
   utilityCheck,
+  utilityGridView,
```

Comment:
Instead of adding a new icon, use the existing as `utilityGridView: navigationApps`.